apps/api/src/server/applications/examination-timetable-items/examination-timetable-items.routes.js

```diff
@@ -33,16 +33,16 @@ router.get(
 	'/:id',
 	/*
         #swagger.tags = ['Applications']
-        #swagger.path = '/applications/examination-timetable-items'
-        #swagger.description = 'Gets all examination timetable items for case'
+        #swagger.path = '/applications/examination-timetable-items/{id}'
+        #swagger.description = 'Gets examination timetable item by id'
         #swagger.parameters['id'] = {
             in: 'path',
 			description: 'Examination timetable item ID',
 			required: true,
 			type: 'integer'
         }
         #swagger.responses[200] = {
-            description: 'List of examination timetable items',
+            description: 'Examination timetable item',
             schema: { $ref: '#/definitions/ExaminationTimetableItemsResponseBody' }
```

Comment:
the response schema definition is now singular, this needs to change as well: Item not Items